### DISCUSSION TITLE
Outdated version notice

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1355] [Q&A] [answered]
is this on-screen message a new thing in nightly builds?
![image](https://github.com/scripthookvdotnet/scripthookvdotnet/assets/121039033/cdc1127d-f656-4d0f-980b-9b86a36ea77d)

**Comment by kagikn** [comment, marked as answer]
yes, not included in v3.6.0 (see 4f4678a for the idea behind)

**Reply by Michael21107** [reply]
thanks
